docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -0,0 +1,200 @@
+</Details>
+
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up Teleport to access your AWS Keyspaces (Apache Cassandra)..
+- Connect to your database through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redis Self-Hosted](../../../img/database-access/guides/cassandra_keyspaces_slefhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redis Cloud](../../../img/database-access/guides/cassandra_keyspaces_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- AWS Account with AWS Keyspaces database and permissions to create and attach IAM policies
+- cqlsh Cassandra client installed and added to your system's `PATH` environment variable.
+- A host, e.g., an Amazon EC2 instance, where you will run the Teleport Database Service.
+
+See [Installation](../../installation.mdx) for details.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:443 \
+  --name=keyspaces \
+  --protocol=cassandra \
+  --aws-account-id=12345678912 \
+  --aws-region=us-east-1 \
+  --labels=env=dev
+```
+
+<Admonition type="note">
+
+The `--auth-server` flag must point to the Teleport cluster's Proxy Service
+endpoint because the Database Service always connects back to the cluster over a
+reverse tunnel.
+
+</Admonition>
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Cloud tenant:
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=mytenant.teleport.sh:443 \
+  --name=keyspaces \
+  --protocol=cassandra \
+  --aws-account-id=12345678912 \
+  --aws-region=us-east-1 \
+  --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+
+## Step 3/5. [Optional] Create AWS Keyspaces role:
+Go to the IAM -> Access Management -> [Roles](https://console.aws.amazon.com/iamv2/home#/roles).
+Press Create Role.
+
+![Create Role Step 1](../../../img/database-access/guides/keyspaces/create-role-step1.png)
```

Comment:
hmm, the create role page looks a different for me
<img width="900" alt="Screenshot 2022-10-27 at 9 36 21 AM" src="https://user-images.githubusercontent.com/12417665/198311589-3effa666-7b08-44e5-ab8b-a2730d08f890.png">
Not a big deal, but i assume i am seeing a newer version which is slightly more user-friendly where you can select your own account.